jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -89,7 +90,6 @@ private Tracer(
 
     this.version = loadVersion();
 
-    Map<String, Object> tags = new HashMap<String, Object>();
```

Comment:
I think it is better to create a new map and add all tags from the parameter `tags`, rather than taking the ownership of the map created outside of the constructor.